### DISCUSSION TITLE
Contract designer to design custom icons for Lucide

### DISCUSSION THREAD

**Discussion by vineetdigit** [lucide-icons/lucide#745] [Q&A] [answered]
Folks,

I've been using feather/lucide extensively and occasionally I need custom icons designed. Is there a recommendation for a contractor who can design custom icons and contribute them to lucide open source? If there's someone, I'd like to connect with them.

thanks,

**Comment by ericfennis** [comment]
@vineetdigit If there icons are *not* too specific and are usable for other projects. You can just open an issue. For other icons you can connect with me.

**Comment by vineetdigit** [comment, marked as answer]
Thanks for your reply. I will write down my requirements for your review and send them next week. You can advise me on how to go about it from there.
